packages/typescript-estree/src/convert.ts

```diff
@@ -1917,7 +1917,13 @@ export class Converter {
         return result;
       }
 
-      case SyntaxKind.TaggedTemplateExpression:
+      case SyntaxKind.TaggedTemplateExpression: {
```

Comment:
Is this meant to be tested some other way because codecov is complaining about it...